create_aio_app/template/{{cookiecutter.project_name}}/setup.py

```diff
@@ -25,24 +24,11 @@ def read_version():
             raise RuntimeError(msg)
 
 
-install_requires = [
-    'aiohttp',
-    'aiohttp_jinja2',
-    'trafaret_config',
-    'markdown2',
-    {%- if cookiecutter.use_postgres == 'y' %}
-    'aiopg[sa]',
-    'psycopg2-binary',
-    'alembic',
-    {%- endif %}
-    {%- if cookiecutter.use_redis == 'y' %}
-    'aioredis',
-    {%- endif %}
-    # todo: remove test requirements
-    'pytest',
-    'pytest-cov',
-    'pytest-aiohttp',
-]
+def read_requirements():
+    my_file = Path("requirements/production.txt")
```

Comment:
The better way for using `Path` it's declaring the `parent` path and use that in a file. The example u can see in this file.
https://github.com/aio-libs/create-aio-app/blob/master/setup.py#L9

```python
parent = pathlib.Path(__file__).parent

def function(f: str) -> str:
    file_path = parent / f

    with open(file_path) as f:
        return f.read().strip()
```

Reply:
also will be better if u use annotation for your function